cli/tests/stake.rs

```diff
@@ -135,8 +135,6 @@ fn test_stake_delegation_force() {
 
 #[test]
 fn test_seed_stake_delegation_and_deactivation() {
-    solana_logger::setup();
-
```

Comment:
I've reverted the changes in this file already, just haven't pushed because I'm still holding out hope for CI.